memcacheinspector/__init__.py

```diff
@@ -64,8 +64,8 @@ def __ne__(self, other):
         return not self.equals(other)
 
 
-_RE_SLAB_STAT = re.compile(b'^STAT (?P<id>\d+):chunk_size (\d+)$')
-_RE_ITEM = re.compile(b'^ITEM (?P<key>\S+) \[(?P<size>\d+) b; (?P<expiration>\d+) s\]$')
+_RE_SLAB_STAT = re.compile(b'^STAT (?P<id>\\d+):chunk_size (\\d+)$')
+_RE_ITEM = re.compile(b'^ITEM (?P<key>\\S+) \\[(?P<size>\\d+) b; (?P<expiration>\\d+) s\\]$')
```

Comment:
A small tip: you can put `r` before a string to mean "raw" and it works well with regexes.  It can be used with `b` for bytes at the same time too.  And then it means you don't have to double-escape the backslashes.  What you've got here is totally good too though.